SETUP.md

```diff
@@ -1,43 +1,57 @@
-   1. Install Docker using the package repository, [link](https://docs.docker.com/engine/install/fedora/#install-using-the-repository).
-   2. Do the post install for _Docker_, [link](https://docs.docker.com/engine/install/linux-postinstall/)
-2. Install _Docker Desktop_, [link](https://docs.docker.com/desktop/linux/install/fedora/).
-   1. Download the _RPM_ package, [link](https://docs.docker.com/desktop/release-notes/).
-3. Enable _Kubernetes_ on _Docker_.
-   1. Start _Docker Desktop_.
-   2. Click on the _Gear_ icon (i.e., settings).
-   3. Click on the _Kubernetes_ menu item.
-   4. Check the _Enable Kubernetes_ checkbox.
-   5. Click the _Apply & Restart_ button.
-4. Register for a _Docker Hub_ account and activate your account, [link](https://hub.docker.com/).
-5. Login to _Docker_, [link](https://docs.docker.com/desktop/linux/#credentials-management).
-6. Install _docker-credential-pass_, [link](https://github.com/docker/docker-credential-helpers/releases).
-   1. Download the appropriate tarball for your machine and extract it.
-   2. `sudo cp /path/to/docker-credential-pass /bin/docker-credential-desktop`
-7. Install _kubectl_, [link](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/).
-   1. Ensure that _kubectl_ is pointing to the Docker Desktop context:
-      1. `kubectl config get-contexts`
-      2. `kubectl config use-context docker-desktop`
+After installing _Docker_ and enabling _Kubernetes_, register for a _Docker Hub_ account, [link](https://hub.docker.com/).
+
+## Fedora Linux
+
+1. Install [_Docker Engine_](https://docs.docker.com/engine/install/fedora/).
+2. Carry out the post install steps, [link](https://docs.docker.com/engine/install/linux-postinstall/).
+3. Install [_kubectl_](https://docs.docker.com/desktop/kubernetes/).
+
+Alternatively, simply install [_Docker Desktop_](https://docs.docker.com/desktop/linux/install/fedora/) and enable [_Kubernetes_](https://docs.docker.com/desktop/kubernetes/).
+
+## MacOS
+
+1. Install [_Docker Desktop_](https://docs.docker.com/desktop/mac/install/).
+2. Enable _Kubernetes_ via _Docker Desktop_, [link](https://docs.docker.com/desktop/kubernetes/).
+
+# Installing Java 8
+
+Installing _Java 8_ can vary depending on your operating system and chosen methodology. The recommended version to install is _OpenJDK 8_.
+
+## Fedora Linux
+
+Installing via _DNF_ is the recommended method for Fedora Linux. Instructions for that can be found [here](https://docs.fedoraproject.org/en-US/quick-docs/installing-java/).
+
+## Mac OS
+
+Installing via [_Homebrew_](https://brew.sh/) is the recommended method for Mac, though it can be used for other operating systems as well. To install _OpenJDK 8_ on a machine with an Intel chip, go [here](https://formulae.brew.sh/formula/openjdk@8). _OpenJDK 8_ is not available for the M1 chipset. So in that case, _OpenJDK 11_ can be used, [link](https://formulae.brew.sh/formula/openjdk@11#default).
```

Comment:
> _OpenJDK 8_ is not available for the M1 chipset.

OpenJDK 8 is preferable in any case. On M1 chipset one can install AdoptOpenJDK or Azul Zulu build.

![image](https://user-images.githubusercontent.com/4959931/172591040-57161b57-5abb-46ae-bc5d-c2e954d6108e.png)

Reply:
As discussed internally, we would like to bump docs to Java 11 :+1:

Reply:
Updated the docs with install instructions for Java 11.